samples/drivers/opamp/output_measure/src/main.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef MAIN_H
+#define MAIN_H
+
+#include <zephyr/drivers/opamp.h>
+
+#define OPAMP_SUPPORT_PROGRAMMABLE_GAIN				\
+	DT_PROP_OR(DT_PHANDLE(DT_PATH(zephyr_user), opamp),	\
+		support_programmable_gain, 0)
+
+#if OPAMP_SUPPORT_PROGRAMMABLE_GAIN
+#if defined(CONFIG_BOARD_FRDM_MCXN947) ||	\
```

Comment:
Boards specific defines shall not be the part of sample-code. Ideally sample code shall not contain any SoC specific dependency.